tests/integration/test_analysisd/test_syscollector/test_syscollector_integration.py

```diff
@@ -0,0 +1,100 @@
+import os
+import pytest
+
+from wazuh_testing.tools.configuration import get_test_cases_data
+from wazuh_testing.tools import (ANALYSISD_QUEUE_SOCKET_PATH, ALERT_FILE_PATH)
+from wazuh_testing.analysis import CallbackWithContext, callback_check_syscollector_alert
+
+pytestmark = [pytest.mark.server]
+
+# Generic vars
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+TEST_RULES_PATH = os.path.join(TEST_DATA_PATH, 'rules')
+local_internal_options = {'analysisd.debug': '2'}
```

Comment:
Why didn't has defined these values on the `wazuh_testing/modules/analisisd/__init__.py` file?

Reply:
Those are values for the test scenario, if we need to move them we could send them to the __init__.py from this test module